File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsBehaviorFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsBehaviorFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsBehaviorFragment().getTitleResourceId()).isEqualTo(R.string.behavior);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsBehaviorFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_behavior);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsBehaviorFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsBehaviorFragmentTest {
    @Test
    fun getTitleResourceId() {
        assertThat(SettingsBehaviorFragment().getTitleResourceId()).isEqualTo(R.string.behavior)
    }

    @Test
    fun getXmlResourceFile() {
        assertThat(SettingsBehaviorFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_behavior)
    }
}
